### ISSUE TITLE
deploy task

### ISSUE THREAD

**Issue by psimyn** [psimyn/ghpbp#3]
squash, commit to gh-pages and push

https://help.github.com/articles/creating-project-pages-manually/

**Comment by choonkending** [comment]
@psimyn Not exactly sure how much automation this should do.

I envision this to be a bash script, that:
- accepts the remote repository of your choice as an argument
- creates the orphan branch for you
- builds the page
- pushes the compiled code to the repo

Why do we need to squash commits?

Not sure if I am completely missing the point?

**Comment by psimyn** [comment]
true, don't need to squash. I think the remote repo is just this repo (i.e. you clone it, then set new remote). So deploy just means push gh-pages branch to origin
